### PR TITLE
Update innr.ts

### PR DESCRIPTION
**Pull request by z647230**
added support for innr sp-240 device:

        zigbeeModel: ['SP 240'],
        model: 'SP 240',
        vendor: 'Innr',

### CODE REVIEW

File: src/devices/innr.ts [hunk 1]
```diff
@@ -684,6 +685,30 @@ const definitions: Definition[] = [
             }
         },
     },
+    {
```
Comment (Koenkk) [inline src/devices/innr.ts:688]: Can you try with the following instead:

```
{
        zigbeeModel: ['SP 240'],
        model: 'SP 240',
        vendor: 'Innr',
        description: 'Smart plug',
        extend: [
            onOff(),
            electricityMeter(),
        ],
    },
```

**Comment by heresiarch** [comment]
You are so fast unbeliebable, thank you @Koenkk  . In the meantime I tried this successfully as workaround with a custom converter:
```javascript
const fz = require('zigbee-herdsman-converters/converters/fromZigbee');
const tz = require('zigbee-herdsman-converters/converters/toZigbee');
const exposes = require('zigbee-herdsman-converters/lib/exposes');
const reporting = require('zigbee-herdsman-converters/lib/reporting');
//const extend = require('zigbee-herdsman-converters/lib/extend');
const {light, onOff, electricityMeter} = require('zigbee-herdsman-converters/lib/modernExtend');
//const e = exposes.presets;
//const ea = exposes.access;

const definition = {
zigbeeModel: ['SP 240'], // The model ID from: Device with modelID 'lumi.sens' is not supported.
model: 'SP 240', // Vendor model number, look on the device for a model number
vendor: 'inrr', // Vendor of the device (only used for documentation and startup logging)
description: 'smart plug innr', // Description of the device, copy from vendor site. (only used for documentation and startup
extend: [
        onOff(),
        electricityMeter({current: {divisor: 1000}, voltage: {divisor: 1}, power: {divisor: 1}, energy: {divisor: 100}}),
        ],
//ota: ota.zigbeeOTA,
};

module.exports = definition;

```

File: src/devices/innr.ts (hunk 1 above)
Comment (z647230) [reply src/devices/innr.ts:688]: it works but...
looking at the "exposes" screen of the home assistant/zigbee2mqtt it shows adjusters on the right side of the screen as if you could change voltage.  (see attachment)
innrsp240exposescreen

The way I implemented/tested your sample is by converting it into an external converter. (see attachment 2)
innrsp240

(ps. last time I programmed was in old fashioned assembler, Cobol and later in C. Now learning on how to work with all the modern stuff ;-) )

**Comment by Koenkk** [comment]
what error message does show up?

File: src/devices/innr.ts (hunk 1 above)
Comment (Koenkk) [reply src/devices/innr.ts:688]: Can you try with:

```js
{
        zigbeeModel: ['SP 240'],
        model: 'SP 240',
        vendor: 'Innr',
        description: 'Smart plug',
        extend: [
            onOff({powerOnBehavior: false}),
            electricityMeter({current: {divisor: 1000}, voltage: {divisor: 1}, power: {divisor: 1}, energy: {divisor: 100}}),
        ],
    },
```

Comment (z647230) [reply src/devices/innr.ts:688]: Configure goes ok, the powerOnBehavior i tried with and without your suggestion as it already worked. But the function to disable, like in your example, works. So i tested with poweronbehavior enabled and all settings off/on/toggle/restore work.
I noticed that the default Zigbee icon also is replaced by a plug jpeg image.
i think that makes it pretty complete unless you want to test other stuff on it ?

**Comment by Koenkk** [comment]
Thanks!

**Comment by z647230** [comment]
You’re welcome.But like said before, the powerInBehavior does work correctly so no need to disable it.Rgdz, MartinGrtz, MartinOn 10 Dec 2023, at 20:50, Koen Kanters ***@***.***> wrote:﻿
Thanks!

**Comment by Koenkk** [comment]
It's disabled: https://github.com/Koenkk/zigbee-herdsman-converters/blob/c7ee83322f9cd4f5c83da7f042e2a5ae5468ec09/src/devices/innr.ts#L668

**Comment by z647230** [comment]
Yes, I noticed but it should be enabled as the powerOnBehavior is supported correctly by the device.I fully tested it with all the states.Grtz, MartinOn 11 Dec 2023, at 19:22, Koen Kanters ***@***.***> wrote:﻿
It's disabled: https://github.com/Koenkk/zigbee-herdsman-converters/blob/c7ee83322f9cd4f5c83da7f042e2a5ae5468ec09/src/devices/innr.ts#L668

**Comment by Koenkk** [comment]
I see, enabled it!

Changes will be available in the [dev branch](https://www.zigbee2mqtt.io/advanced/more/switch-to-dev-branch.html) in a few hours from now.

**Pull request merged**